src/components/Navigator/Navigator.tsx

```diff
@@ -1,19 +1,122 @@
 import React, { useEffect, useState } from "react";
 import { styled } from "@stitches/react";
+import NavigatorItem from "components/NavigatorItem/NavigatorItem";
+import sampleParsedVtt from "../../manifests/sampleParsedVtt";
```

Comment:
My VSCode is complaining about this line...maybe because of the .json file extension?  We should see if we can match up our TypeScript linting between our IDEs?

Reply:
@adamjarling Yeah let's look. Line 4 here won't stay to production, but in the case of importing other JSON files it'd be smart to see how to best handle.